docs/install/from_source.rst

```diff
@@ -73,6 +73,16 @@ linux operating systems, execute (in a terminal):
     sudo apt-get update
     sudo apt-get install -y python3 python3-dev python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev
 
+On macOS, whether using an Intel or M1 Mac we can use homebrew. Make sure to follow the post installation steps for each of the below commaands.
+Furthermore, avoid using python 3.9.X+ which is not supported at the time of this writing. With this, you should have the ability to compile a 
```

Comment:
is there a link to a bug stating the problem?

Reply:
Looking this issue up online, it seems like support was added earlier this year, so information is conflicting.

Reply:
Eh there definitely are still issues however since 3.9 changed the python IR which TVM apparently manipulates in some parts.

Reply:
Huh, if it's a general problem, we should state that as part of the overall requirements. I don't mind having that be part of this patch, and if there is more information we need to file a bug against TVM.

Reply:
Yep, I personally don't have enough background to make a good issue. But maybe others do @areusch @tkonolige

Reply:
https://github.com/apache/tvm/issues/8577 <-- made a dummy issue to link against in the meantime. Made some changes to the wording to make it a general requirement